values/values.go

```diff
@@ -145,19 +145,14 @@ var (
 	// InvalidValue is a non nil value who's type is semantic.Invalid
 	// TODO (algow): create a invalid monotype
 	InvalidValue = value{t: semantic.BasicBool}
-
-	// Null is an untyped nil value.
-	// TODO (algow): create a null monotype
-	Null = value{t: semantic.BasicBool}
 )
 
 // New constructs a new Value by inferring the type from the interface. If the interface
 // does not translate to a valid Value type, then InvalidValue is returned.
```

Comment:
Can you update the comment to indicate this method can panic?

Reply:
Yes I can.